src/components/DeliverableAnalysisDetailPage/DeliverableAnalysisDetailPage.tsx

```diff
@@ -62,13 +81,30 @@ export const DeliverableAnalysisDetailPage = () => {
         onSuccess: (res) => {
           const versionId = res.response.data.productVersion?.id;
           if (versionId) {
-            serviceContainerProductVersionRunner({ serviceData: { id: versionId } });
+            serviceContainerProductVersionRunner({
+              serviceData: { id: versionId },
```

Comment:
Please avoid this unrelated formatting change, the original formatting can be used.